packages/cli/src/modules/insights/__tests__/insights.service.test.ts

```diff
@@ -664,3 +671,105 @@ describe('getMaxAgeInDaysAndGranularity', () => {
 		});
 	});
 });
+
+describe('shutdown', () => {
+	let insightsService: InsightsService;
+
+	const mockCollectionService = {
+		shutdown: jest.fn().mockResolvedValue(undefined),
+		stopFlushingTimer: jest.fn(),
+	} as unknown as InsightsCollectionService;
+
+	const mockCompactionService = {
+		stopCompactionTimer: jest.fn(),
+	} as unknown as InsightsCompactionService;
```

Comment:
I notice these are using a test DB so are meant as integration - should we be mocking? I never had full clarity on this.

Reply:
hum, good question, that I guess we would need to reach a consensus on. For Insights module, we have multiple tests with real data in all service tests (insights.service.test.ts, insights-collection.service.test.ts, insights-compaction.service.test.ts). In my mind, integration tests in the repo were rather more backend e2e tests, but it's up to discussion